wherehows-dao/src/main/java/wherehows/dao/DatasetsDao.java

```diff
@@ -85,14 +85,12 @@
   private static final String GET_DATASAT_COMPLIANCE_BY_DATASET_ID =
       "SELECT dataset_id, dataset_urn, compliance_purge_type, compliance_entities, confidentiality, "
           + "dataset_classification, field_classification, record_owner_type, retention_policy, "
-          + "geographic_affinity, modified_by, modified_time "
-          + "FROM dataset_compliance WHERE dataset_id = ?";
+          + "geographic_affinity, modified_by, modified_time " + "FROM dataset_compliance WHERE dataset_id = ?";
```

Comment:
In this case, don't do reformat, as the "FROM" sentence is more clear at the next line.